ssp

```diff
@@ -2505,8 +2514,28 @@ sub print_ftpserver_info {
 
     my $passivetext = $passiveports ? "enabled - " . ( $fwppactive ? "allowed in iptables" : "not found in iptables" ) : "not enabled";
 
+    my $passiveiptext = "";
+
+    if ($passiveip) {
+        if ( $passiveip ne $EXTERNAL_IP_ADDRESS ) {
+            if ( defined( $CPCONF{'ftpserver'} ) ) {
+                if ( $CPCONF{'ftpserver'} eq 'proftpd' ) {
+                    $passiveiptext = "MasqueradeAddress ( " . $passiveip . " ) doesn't match license IP";
```

Comment:
Concatenate this text on the end of $passivetext, it's not necessary to put it in a separate $passiveiptext.  6 lines of code can be dropped by doing so, reducing complexity.